aspnetcore/client-side/blazor/introduction/index.md

```diff
@@ -0,0 +1,170 @@
+
+![AOT mode](index/_static/aotmode.png)
+
+This is similar to how the ngen tool has historically allowed AOT compilation of .NET binaries into native machine code. More recently, CoreRT provides a complete native AOT .NET runtime for executing .NET binaries.
+
+## Interpreted versus AOT
+
+Interpreted mode provides a faster development cycle than AOT. When code is changed under the interpreted model, the app can be rebuilt and reloaded in the browser within a few seconds. An AOT rebuild might take several minutes to compile and reload. Because interpreted mode offers faster recompilation, it might become the default mode for compilation during development. More work is required before a conclusion can be drawn on how to best apply interpreted and AOT modes of compilation with WebAssembly.
+
+## Blazor, a SPA framework
+
+Blazor has a growing standard feature set to solve common app requirements, such as UI composition, state management, and routing. Features are designed around the strengths of .NET and the C# language with careful consideration given to tooling support.
+
+Blazor is inspired by today's top SPA frameworks such as React, Vue, and Angular, plus some Microsoft technologies, such as Razor Pages. The goal is to provide SPA features that web developers have found most successful in existing frameworks in a way that capitalizes on .NET's strengths.
+
+## Blazor components
+
+In SPA frameworks, apps are built with *components*. A component usually represents a piece of UI, such as a page, dialog, or data entry form. Components can be nested, reused, and shared between projects.
+
+In Blazor, a component is a .NET class. The class can either be written directly, as a C# class (*\*.cs*), or more commonly in the form of a Razor markup page (*\*.cshtml*).
+
+Razor, which has been around since 2010, is a syntax for combining HTML markup with C# code. Razor is designed for developer productivity, allowing the developer to switch between markup and C# in the same file with IntelliSense support. Here's an example of how to express a simple custom dialog component in a Razor file:
+
+```cshtml
+<div>
+    <h2>@Title</h2>
+    @RenderContent(Body)
+    <button onclick=@OnOK>OK</button>
+</div>
+
+@functions {
+    public string Title { get; set; }
+    public Content Body { get; set; }
+    public Action OnOK { get; set; }
+}
+```
+
+When this component is used elsewhere in the app, IntelliSense speeds development with syntax completion and parameter info.
+
+Many design patterns are possible with this simple foundation. This includes patterns recognizable in other SPA frameworks, such as stateful components, functional stateless components, and higher-order components.
```

Comment:
~simple~ is a banned word. you can use basic.
What foundation? Razor? Say so.

> This includes patterns 

What is this?

Reply:
I'm fixing this, but I really don't have a clear, specific picture (and I doubt many readers will either) of what these components are. I can infer what they mean, but it's hard to pin it down. I'll see if I can find something to link for each. Otherwise, we're better off just cutting this sentence and moving the first sentence up. I'll see what I can do with it.

Reply:
Yep ... we don't have anything. I'm cutting the 2nd sentence and moving the first sentence up to where Razor is introduced. I'm also adding a link to our Razor topic.